avocado/plugins/run.py

```diff
@@ -157,6 +157,13 @@ def configure(self, parser):
                                      'will be kept in the test suite found '
                                      'previously to filtering.'))
 
+        parser.add_argument("--list-test-data-directories", action="store_true",
```

Comment:
It goes into the `test` log so I think more appropriate name would be `--log-test-data-directories`.